components/proposalSignForm.tsx

```diff
@@ -202,7 +202,11 @@ function ProposalSignForm({
               ))}
             </div>
             {!!proposal.ui.content.find(
-              v => "addOwners" in v || "removeOwners" in v
+              v =>
+                "addOwners" in v ||
+                "removeOwners" in v ||
+                "changeThreshold" in v ||
+                "adjustExpirationPeriod" in v
             ) && (
               <span className="mt-2 text-xs font-light text-yellow-500">
                 This proposal will update the signers for all the active
```

Comment:
```suggestion
                This proposal will update the setting for all the active
```
in case, it's not a case of adding/removing owners.